lib/pbench/server/api/resources/server_configuration.py

```diff
@@ -226,6 +238,6 @@ def _put(self, params: ApiParams, _) -> Response:
         """
 
         if params.uri:
-            return self._put_key(params)
+            return self._put_key(params, context)
         else:
-            return self._put_body(params)
+            return self._put_body(params, context)
```

Comment:
Alternately, perhaps,
```
        func = self._put_key if params.uri else self._put_body
        func(params, context)
```
(I think I'm getting bored of this review. 🥱)

Reply:
Maybe, but ugly and not easier to read or even much more compact.

Wow ... *bored*. I guess I should try to be more exciting in the future?

Reply:
Less is more.  😉